compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/AnnotationDesugar.java

```diff
@@ -300,35 +300,37 @@ private void attachSchedulerPolicy(BLangFunction function) {
             if (!annotation.annotationName.value.equals("strand")) {
                 continue;
             }
-            List<RecordLiteralNode.RecordField> fields = ((BLangRecordLiteral) annotation.expr).fields;
-            for (RecordLiteralNode.RecordField field : fields) {
-                if (field.getKind() != NodeKind.RECORD_LITERAL_KEY_VALUE) {
-                    continue;
-                }
-                BLangRecordLiteral.BLangRecordKeyValueField keyValue =
-                        (BLangRecordLiteral.BLangRecordKeyValueField) field;
-                BLangExpression expr = keyValue.key.expr;
-                if (expr.getKind() != NodeKind.SIMPLE_VARIABLE_REF) {
-                    continue;
-                }
-                BLangIdentifier variableName = ((BLangSimpleVarRef) expr).variableName;
-                if (variableName.value.equals("name")) {
-                    if (keyValue.valueExpr.getKind() != NodeKind.LITERAL) {
+            if (annotation.expr != null) {
```

Comment:
We can probably check the inverse and return/continue early to avoid the indentation in the block, right?